modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -355,6 +357,10 @@ SHOW ALIAS `northwind-2022` FOR DATABASE YIELD name, properties
 A database alias can target a remote database by providing an URL and the credentials of a user on the remote Neo4j DBMS.
 See xref:database-administration/aliases/remote-database-alias-configuration.adoc[] for the necessary configurations.
 
+Since remote database aliases target databases that are not in this DBMS, they do not fetch the default language from their target like the local database aliases.
+Instead, remote database aliases are assigned the default language given by xref:configuration/configuration-settings.adoc#config_db.query.default_language[`db.query.default_language`] on creation.
+There is currently no command to set it explicitly nor change it on existing remote database aliases.
```

Comment:
the commands have now been merged so we should update this and the show alias parts to include the updates around it